### PR TITLE
RiotGames: add detection of flows

### PR DESCRIPTION
**Pull request by JaCk00346**

### CODE REVIEW

File: src/lib/ndpi_content_match.c.inc [hunk 1]
```diff
@@ -1344,6 +1344,13 @@ static ndpi_protocol_match host_match[] =
    { "metrika.yandex.",                  "YandexMetrika", NDPI_PROTOCOL_YANDEX_METRIKA, NDPI_PROTOCOL_CATEGORY_WEB, NDPI_PROTOCOL_SAFE, NDPI_PROTOCOL_DEFAULT_LEVEL },
    { "direct.yandex.",                   "YandexDirect", NDPI_PROTOCOL_YANDEX_DIRECT, CUSTOM_CATEGORY_ADVERTISEMENT, NDPI_PROTOCOL_TRACKER_ADS, NDPI_PROTOCOL_DEFAULT_LEVEL },
 
+   { "riotgames.com",                    "RiotGames", NDPI_PROTOCOL_RIOTGAMES, NDPI_PROTOCOL_CATEGORY_GAME, NDPI_PROTOCOL_FUN, NDPI_PROTOCOL_DEFAULT_LEVEL},
+   { "pvp.net",                          "RiotGames", NDPI_PROTOCOL_RIOTGAMES, NDPI_PROTOCOL_CATEGORY_GAME, NDPI_PROTOCOL_FUN, NDPI_PROTOCOL_DEFAULT_LEVEL},
+   { "a17.d.a.kamai.net",                "RiotGames", NDPI_PROTOCOL_RIOTGAMES, NDPI_PROTOCOL_CATEGORY_GAME, NDPI_PROTOCOL_FUN, NDPI_PROTOCOL_DEFAULT_LEVEL},
+   { "a11.d.akamai.net",                 "RiotGames", NDPI_PROTOCOL_RIOTGAMES, NDPI_PROTOCOL_CATEGORY_GAME, NDPI_PROTOCOL_FUN, NDPI_PROTOCOL_DEFAULT_LEVEL},
+   { "a248.e.akamai.net",                "RiotGames", NDPI_PROTOCOL_RIOTGAMES, NDPI_PROTOCOL_CATEGORY_GAME, NDPI_PROTOCOL_FUN, NDPI_PROTOCOL_DEFAULT_LEVEL},
```
Comment (IvanNardi) [inline src/lib/ndpi_content_match.c.inc]: These 3 domains seems to be generic Akami domains to me... Am I wrong?

File: src/lib/ndpi_content_match.c.inc [hunk 2]
```diff
@@ -1478,6 +1485,8 @@ static ndpi_tls_cert_name_match tls_certificate_match [] = {
   { "CN=cloudflareclient.com",         NDPI_PROTOCOL_CLOUDFLARE_WARP },
   { "O=V Kontakte LLC",                NDPI_PROTOCOL_VK },
   { "O=Yandex LLC",                    NDPI_PROTOCOL_YANDEX },
+  { "O=Riot Games, Inc.",              NDPI_PROTOCOL_RIOTGAMES   },
+  { "O=Riot Games Inc",                NDPI_PROTOCOL_RIOTGAMES   },
```
Comment (IvanNardi) [inline src/lib/ndpi_content_match.c.inc:1486]: Could you attach a trace matching these two rules, please?

**Review by IvanNardi — CHANGES_REQUESTED** [review]

File: src/lib/ndpi_content_match.c.inc (hunk 2 above)
Comment (JaCk00346) [reply src/lib/ndpi_content_match.c.inc:1486]: [certificate.zip](https://github.com/ntop/nDPI/files/11181242/certificate.zip)

File: src/lib/ndpi_content_match.c.inc (hunk 1 above)
Comment (JaCk00346) [reply src/lib/ndpi_content_match.c.inc]: i added these domains by mistake

**Pull request merged**